scripts/appcatalog.sh

```diff
@@ -4,18 +4,31 @@ set -ex
 
 ./appcatalog $* &
 
-[ "x${IMPORT_SAMPLE_DATA}" != "x" ] && {
+wait_for_apis () {
     export API_BASE_URL=http://localhost:8081/api
     # wait for api
     ret=1
     while [ $ret -ne 0 ]
```

Comment:
Can we limit the waiting to 60 seconds ?

Reply:
Yes but we must abort the container no ?

Reply:
My be in a next PR ... but for me we must wait infinitely